docs/components/zeebe/open-source/get-help-get-involved.md

```diff
@@ -1,19 +1,20 @@
 ---
 id: get-help-get-involved
-title: "Get help & get involved"
+title: "Get help and get involved"
+description: "Ask questions, report problems, and make contributions."
 ---
 
 We provide a few different public-facing Zeebe support and feedback channels where users can ask questions, report problems, and make contributions.
 
 ### Camunda Cloud user forum
 
-The best place to ask questions about Zeebe and to troubleshoot issues is the [forum](https://forum.camunda.io).
+The best place to ask questions about Zeebe and troubleshoot issues is the [forum](https://forum.camunda.io).
 
 The Zeebe team monitors the forum closely, and we do our best to respond to all questions in a timely manner.
 
 ### Public Slack group
 
-Join our [public Slack group](https://zeebe-slack-invite.herokuapp.com/), where you can ask one-off questions, share community contributions, and connect with other Zeebe users.
+Join our [public Slack group](https://zeebe-slack-invite.herokuapp.com/), where you can ask questions, share community contributions, and connect with other Zeebe users.
 
 ### Create an issue in GitHub
```

Comment:
I don't prefer it but we also don't have any working process for community issue right now. Yes all issues are arriving in the Zeebe project. Maybe once we new "empty" repository which represent all cloud components we could use that issue tracker, but again without a process and somebody feeling responsible for it, it will also just be a blackhole. So for now our only maintained issue tracker is the one from Zeebe.